src/composables/useWindowingConfigInitializer.ts

```diff
@@ -1,13 +1,42 @@
 import { useImage } from '@/src/composables/useCurrentImage';
 import { useWindowingConfig } from '@/src/composables/useWindowingConfig';
 import { WLAutoRanges, WL_AUTO_DEFAULT, WL_HIST_BINS } from '@/src/constants';
-import { useDICOMStore } from '@/src/store/datasets-dicom';
+import { getWindowLevels, useDICOMStore } from '@/src/store/datasets-dicom';
 import useWindowingStore from '@/src/store/view-configs/windowing';
 import { Maybe } from '@/src/types';
 import type { TypedArray } from '@kitware/vtk.js/types';
 import { watchImmediate } from '@vueuse/core';
 import { MaybeRef, computed, unref, watch } from 'vue';
 
+// Original source from https://www.npmjs.com/package/compute-range and vtkDataArray
+// Modified to assume one component array
+function fastComputeRange(arr: number[] | TypedArray) {
```

Comment:
This function is accessible via `vtkDataArray.fastComputeRange`. You should be able to just use that + set numComponents to 1.

Reply:
Fixed